dockers/base-cuda/Dockerfile

```diff
@@ -47,6 +48,8 @@ RUN \
         ca-certificates \
         software-properties-common \
         libopenmpi-dev \
+        openmpi-bin \
+        ssh \
```

Comment:
We need these packages to avoid the errors pasted in https://github.com/PyTorchLightning/pytorch-lightning/pull/12373#issuecomment-1118241218.